### PR TITLE
CO-5: Configure Minio deployment

### LINKED ISSUE #5: Configure deployment of Minio
https://min.io/
* use docker compose
* set up everything that is required to use storage (create bucket, user, assign roles, give permissions if needed) 
* expose web UI publicly

### PR DESCRIPTION
**Pull request by MurzinR**
Resolves #5

### CODE REVIEW

File: core/env/docker-compose.yml [hunk 1]
```diff
@@ -22,5 +22,16 @@ services:
...
+      - 127.0.0.1:${MINIO_PORT:-0}:9001
+    environment:
+      MINIO_ROOT_USER: cocktail_offerer
+      MINIO_ROOT_PASSWORD:
+    command: server --console-address ":9001" /data
+    volumes:
+      - minio_storage:/data
 volumes:
   postgres:
+  minio_storage:
```
Comment (KirillSmirnow) [inline core/env/docker-compose.yml]: Volume for postgres data is named `postgres`. It's not named `postgres_storage`. So why be inconsistent with it?

File: core/env/docker-compose.yml [hunk 2]
```diff
@@ -22,5 +22,16 @@ services:
       POSTGRES_PASSWORD:
     volumes:
       - postgres:/var/lib/postgresql/data
+  minio:
+    image: minio/minio:RELEASE.2023-02-22T18-23-45Z.fips
+    ports:
+      - 127.0.0.1:${MINIO_PORT:-0}:9001
+    environment:
+      MINIO_ROOT_USER: cocktail_offerer
+      MINIO_ROOT_PASSWORD:
```
Comment (KirillSmirnow) [inline core/env/docker-compose.yml:31]: Variable must be declared in workflow configuration

File: core/env/docker-compose.yml [hunk 3]
```diff
@@ -22,5 +22,16 @@ services:
       POSTGRES_PASSWORD:
     volumes:
       - postgres:/var/lib/postgresql/data
+  minio:
+    image: minio/minio:RELEASE.2023-02-22T18-23-45Z.fips
+    ports:
+      - 127.0.0.1:${MINIO_PORT:-0}:9001
```
Comment (KirillSmirnow) [inline core/env/docker-compose.yml:28]: Variable must be declared in workflow configuration

**Review by KirillSmirnow — CHANGES_REQUESTED** [review]

File: core/env/docker-compose.yml (hunk 1 above)
Comment (MurzinR) [reply core/env/docker-compose.yml]: Fixed

File: core/env/docker-compose.yml (hunk 2 above)
Comment (MurzinR) [reply core/env/docker-compose.yml:31]: added

File: core/env/docker-compose.yml (hunk 3 above)
Comment (MurzinR) [reply core/env/docker-compose.yml:28]: added

File: core/env/docker-compose.yml [hunk 4]
```diff
@@ -22,5 +22,16 @@ services:
...
+    environment:
+      MINIO_ROOT_USER: cocktail_offerer
+      MINIO_ROOT_PASSWORD:
+    command: server --console-address ":9001" /data
+    volumes:
+      - minio
```
Comment (KirillSmirnow) [inline core/env/docker-compose.yml]: Mapping is not complete (`host:container`)

**Review by KirillSmirnow — CHANGES_REQUESTED** [review]

**Review by KirillSmirnow — APPROVED** [review]

**Pull request merged**